mitreattack/navlayers/generators/overview_generator.py

```diff
@@ -0,0 +1,210 @@
+from stix2 import Filter
+from itertools import chain
+try:
+    from ..exporters.matrix_gen import MatrixGen
+    from ..core.exceptions import BadInput, typeChecker, categoryChecker
+    from ..core.layer import Layer
+    from ..generators.gen_helpers import remove_revoked_depreciated, construct_relationship_mapping, \
+        mitre_attack_type_strings
+except ValueError:
+    from mitreattack.navlayers.exporters.matrix_gen import MatrixGen
+    from mitreattack.navlayers.core.exceptions import BadInput, typeChecker, categoryChecker
+    from mitreattack.navlayers.core.layer import Layer
+    from mitreattack.navlayers.generators.gen_helpers import remove_revoked_depreciated, \
+        construct_relationship_mapping, mitre_attack_type_strings
+except ImportError:
+    from navlayers.exporters.matrix_gen import MatrixGen
+    from navlayers.core.exceptions import BadInput, typeChecker, categoryChecker
+    from navlayers.core.layer import Layer
+    from navlayers.generators.gen_helpers import remove_revoked_depreciated, construct_relationship_mapping, \
+        mitre_attack_type_strings
+
+
+class UnableToFindTechnique(Exception):
+    pass
+
+
+class OverviewGenerator:
+    """Generates a Layer file that provides an overview of entities related to each technique"""
+    def __init__(self, source, matrix='enterprise', local=None):
```

Comment:
I think it would be more correct and consistent for the "matrix" argument to be domain since domains can have multiple matrices.

Reply:
I can agree with that.